src/Microsoft.ML.Data/StaticPipe/PipelineColumnAnalyzer.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Runtime;
+using System;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Core.Data;
+
+namespace Microsoft.ML.Data.StaticPipe
+{
+    internal static class PipelineColumnAnalyzer
+    {
+        public static void Analyze<TIn, TOut>(Func<TIn, TOut> func)
```

Comment:
>Analyze [](start = 27, length = 7)

Rename to Initialize

Reply:
/cc @yaeldekel

---
In reply to: [214473601](https://github.com/dotnet/machinelearning/pull/778#discussion_r214473601) [](ancestors = 214473601)